### PR TITLE
Adjust issue template and PR template to fit OpenBao

### PR DESCRIPTION
**Pull request by Scorpil**
Solves #3 

Removing plugin submission template for now.
Added RFC template based on OpenTofu's with slight modifications.

Labels are configured based on #4

### CODE REVIEW

**Comment by jbutlerdev** [comment]
Personally I like bao. It looks like opentofu uses tofu, so it would make sense to follow their lead

**Comment by Scorpil** [comment]
Agree @jbutlerdev. I don't believe there are any popular `bao` packages that could cause a clash on some systems, and being in sync with OpenTofu is a plus.

**Comment by naphelps** [comment]
Make sure all commits use `-s` or `--signoff` for sign off.

File: .github/ISSUE_TEMPLATE/rfc.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        - Are there any disadvantages of implementing this?
+      placeholder:
+      value:
+    validations:
+      required: false
+
+  - type: textarea
+    id: security
+    attributes:
+      label: Security Implications
```
Comment (cipherboy) [inline .github/ISSUE_TEMPLATE/rfc.yml:98]: I don't think its worth highlighting _everything_ in the RFC design, but perhaps adding user experience here would be a good catch-all bucket: e.g., developer experience integrating with this proposed change, operator experience trying to manage/configure it, ...

Just to make sure it gets thought about before dropping a (complex but useful) feature in...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
 
 ---
 
 <!-- Please reserve GitHub issues for bug reports and feature requests.
 
-For questions, the best place to get answers is on our [discussion forum](https://discuss.hashicorp.com/c/vault), as they will get more visibility from experienced users than the issue tracker.
+For questions, the best place to get answers is on our [mailing list](https://lists.lfedge.org/g/openbao), as they will get more visibility from experienced users than the issue tracker.
 
-Please note: We take Vault's security and our users' trust very seriously. If you believe you have found a security issue in Vault, please responsibly disclose by contacting us at security@hashicorp.com. Our PGP key is available at [our security page](https://www.hashicorp.com/security/).
+Please note: We take OpenBao's security and our users' trust very seriously. If you believe you have found a security issue in OpenBao, please responsibly disclose by contacting us at TODO: security email. Our PGP key is available at [TODO: our security page](TODO).
```
Comment (joewxboy) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: Security email is openbao-security@lists.lfedge.org

**Review by joewxboy — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/rfc.yml (hunk 1 above)
Comment (Scorpil) [reply .github/ISSUE_TEMPLATE/rfc.yml:98]: Fair enough, added ✅

**Review by naphelps — APPROVED** [review]
Looks good to me!

**Pull request merged**